### ISSUE TITLE
schema color variables aren't consistent with LESS color variables

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-vanilla#464]
### Subject of the issue
In a previous PR, the [blue colour variable was updated in the LESS to meet AA color contrast](https://github.com/adaptlearning/adapt-contrib-vanilla/pull/328) but the hex codes referencing the blue in the schemas were not updated.

In the AAT, the schema color variables will always take precedence over the LESS variables so the old blue, `#36CDE8` (pre-PR) still applies and fails colour contrast.

### Expected behaviour
I'd suggest removing the `default` hex codes from the schemas as the default Vanilla colours are already applied via the CSS. Keeping the `default` values is another maintenance task when updating the colour variables - both maintaining Vanilla and creating bespoke themes. As per this issue, it's easily forgotten to update the schemas when amending the LESS variables.
